src/spetlr/etl/extractors/__init__.py

```diff
@@ -1,2 +1,3 @@
 from .incremental_extractor import IncrementalExtractor  # noqa: F401
 from .simple_extractor import SimpleExtractor  # noqa: F401
+from .stream_extractor import StreamExtractor  # noqa: F401
```

Comment:
maybe set the `__all__` variable and you can get rid of the `# noqa: F401`. Suggestion:
```python
from .incremental_extractor import IncrementalExtractor
from .simple_extractor import SimpleExtractor
from .stream_extractor import StreamExtractor

__all__ = [
    "IncrementalExtractor",
    "SimpleExtractor",
    "StreamExtractor",
]
```

This could be implemented for all init files in this PR. Again it is just a suggestion